restli-docgen/src/main/java/com/linkedin/restli/docgen/DefaultDocumentationRequestHandler.java

```diff
@@ -236,11 +248,11 @@ private static RoutingException createRoutingError(String path)
   private static final String DOC_VIEW_DOCS_ACTION = "docs";
   private static final String DOC_RESOURCE_TYPE = "rest";
   private static final String DOC_DATA_TYPE = "data";
-  private static final String DOC_JSON_FORMAT = "json";
+  public static final String DOC_JSON_FORMAT = "json";
```

Comment:
Instead of exposing this field, it could be cleaner to lower redundancy by instead using `RestLiDocumentationRenderer.DocumentationFormat.JSON.toString.toLowerCase()`.